sessions/class-01/exercise/start.js

```diff
@@ -10,7 +10,18 @@
  *
  * @returns {number} Value with "number" type, 0 if not a number.
  */
-export function safelyConvertToNumber(possiblyNum) {}
+export function safelyConvertToNumber(possiblyNum) {
+    if(typeof possiblyNum === 'symbol'){
+        return 0;
+    }
+    let number = Number.parseFloat(possiblyNum);
+    if(!Number.isNaN(number)){
+        return number;
+    }else{
+        return 0;
+    }
```

Comment:
This works. But instead of the long statements, we could've just use the ternary expression

```js
return Number.isNaN(number) ? 0 : number;
```